src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -219,4 +222,26 @@ public ResponseEntity<CaseDetails> respondToApplication(
         return ok(finalCaseDetails);
     }
 
+    /**
+     * Updates SendNotification status.
+     *
+     * @param authorization jwt of the user
+     * @param request       the request object which contains sendNotification id and new status value passed
+     *                      from sya-frontend
+     * @return the new updated case wrapped in a {@link CaseDetails}
+     */
+    @PutMapping("/update-notification-state")
+    @Operation(summary = "Update notification state")
+    @ApiResponseGroup
+    public ResponseEntity<CaseDetails> updateSendNotificationState(
```

Comment:
I will approve but I think this should be a new controller. I'd also like to split out the TSE endpoints into a new controller in future.